website/www/site/content/en/documentation/runners/direct.md

```diff
@@ -26,6 +26,7 @@ The Direct Runner executes pipelines on your machine and is designed to validate
 * enforcing encodability of elements
```

Comment:
also considered add the context of these enforcement by javadoc links but running into #24265 (DirectOptions method javadoc are empty as a result)